src/app/view/templates/beol/beol.component.ts

```diff
@@ -154,15 +154,15 @@ export class BeolComponent implements OnInit {
 
         const Du_Fresne = new Correspondent("Du Fresne", "(DE-588)1142274934");
 
-        const Amedee_Francois_Frezier = new Correspondent("Amedee Francois Frezier", "(DE-588)117537373");
+        const Amedee_Francois_Frezier = new Correspondent("Amedee François Frezier", "(DE-588)117537373");
```

Comment:
actually the correct name should be "Amédée François Frézier" according to name index and DNB
https://portal.dnb.de/opac.htm?query=117537373&method=simpleSearch&cqlMode=true

Reply:
done in https://github.com/dhlab-basel/Salsah/pull/206/commits/a1e7e6768121de38743b51670be2e1168d3b6395